eth/chains/base.py

```diff
@@ -315,13 +315,31 @@ def validate_gaslimit(self, header: BlockHeader) -> None:
     def validate_uncles(self, block: BaseBlock) -> None:
         raise NotImplementedError("Chain classes must implement this method")
 
-    @abstractmethod
+    @classmethod
     def validate_chain(
-            self,
+            cls,
             parent: BlockHeader,
```

Comment:
What do you think about the name `anchor` for this to free up the name `parent` name for the ephemeral loop variable.

Reply:
How about `root` and `descendants` instead of `parent` and `children` for the method args?